src/interfaces/ffi/api.rs

```diff
@@ -181,6 +185,15 @@ pub unsafe extern "C" fn h_search(
         )
     };
 
+    let fetch_chains_batch_size = if fetch_chains_batch_size <= 0 {
+        SECURE_FETCH_CHAINS_BATCH_SIZE
+    } else {
+        ffi_unwrap!(
+            usize::try_from(fetch_chains_batch_size),
+            "fetch_chains_batch_size must be a positive int"
```

Comment:
I am not sure this error message is adequate since we treated all negative cases above. Use back-quotes around the parameter name.